src/components/vm/hostdevs/hostDevAdd.scss

```diff
@@ -1,3 +1,9 @@
+@import "../../../lib/_global-variables.scss";
+@import "@patternfly/patternfly/components/Table/table.scss";
+@import "@patternfly/patternfly/components/Table/table-grid.scss";
+
+
+
```

Comment:
@KKoukiou is there any specific reason (in scss) for 3 empty lines after imports, or just a mistake?

Reply:
Just mistake.